lib/formatters/tapFormatter.mjs

```diff
@@ -1,10 +1,10 @@
 import { isNumber } from '../utils/validateTypes.mjs';
 import preprocessWarnings from './preprocessWarnings.mjs';
 
-/** @import {Warning} from 'stylelint' */
+/** @import {Warning, Formatter, LintResult} from 'stylelint' */
```

Comment:
```suggestion
/** @import {Warning, Formatter} from 'stylelint' */
```

I didn't see `LintResult` used, so maybe can be omitted?
Or maybe I overlooked it?